build-in-container/README.md

```diff
@@ -38,8 +41,19 @@ ls out/RPMS/x86_64/
 #  Run the tools manually
 make build-packages SRPM_PACK_LIST="hello_world_demo" -j$(nproc)
 
-# Provide optional arguments
+# Use optional arguments
+## Provide path to Mariner directory. If this option is not used, the current directory is treated as Mariner directory
 ./CBL-MarinerTutorials/mariner-docker-builder.sh -i --mariner_dir /path/to/CBL-Mariner/
+
+## Install RPMs from a custom repo, by providing path to .repo file
+./CBL-MarinerTutorials/mariner-docker-builder.sh -i --RPM_repo_file path/to/custom-repo-file
```

Comment:
rpm/yum/tdnf/dnf are a bit pedantic and explicitly require repo files to end in `.repo`, and our tools I think also look for files the same way. It would be best to show that everywhere to try and reduce possible confusion.

Reply:
Added this to README, and updated all mentions of custom-repo to custom-repo.repo in README